src/main/java/core/basesyntax/writer/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.writer;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String report, String filePath) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(filePath))) {
+            writer.write(report);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write a " + filePath);
```

Comment:
Consider including the original exception `e` in the `RuntimeException` to provide more context for debugging. Also, improve the error message for clarity, e.g., `throw new RuntimeException("Can't write to file " + filePath, e);`